### PR TITLE
feat(backend): Added endpoint to get a movie details (#27)

### PR DESCRIPTION
**Pull request by revodark**

### CODE REVIEW

File: backend/src/controllers/movies.js [hunk 1]
```diff
@@ -13,4 +13,16 @@ const getByCategory = (req, res, next) => {
   }
 };
 
-export default getByCategory;
+const getDetails = (req, res, next) => {
+  const { id } = req.params;
```
Comment (Rolly007) [inline backend/src/controllers/movies.js]: maybe you can do second level destructuring

File: backend/src/modules/services/movies.test.js [hunk 2]
```diff
@@ -33,3 +33,17 @@ describe('Test Movies Axios Controller', () => {
     expect(res).toStrictEqual({ results: getResult });
   });
 });
+
+describe('Test getMovieDetails Axios request', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  test('test getMoviesByCategory function with a successful request', async () => {
```
Comment (Rolly007) [inline backend/src/modules/services/movies.test.js]: Add blank line here please

Comment (revodark) [reply backend/src/modules/services/movies.test.js]: thanks Americo!

File: backend/src/controllers/movies.js (hunk 1 above)
Comment (revodark) [reply backend/src/controllers/movies.js]: thanks! it's solved

**Pull request closed without merging**